app/src/main/res/values-ja/strings.xml

```diff
@@ -2,7 +2,7 @@
 <resources>
     <!-- Japanese -->
     <!-- general -->
-    <string name="is_not_a_valid_connection_string">%1$は有効な接続文字列ではありません！</string>
+    <string name="is_not_a_valid_connection_string">%1$sは有効な接続文字列ではありません！</string>
```

Comment:
Did you mean to change this translation line in this PR?

Reply:
@leto Yes it's typo and throwing a error in the Travis CI.

Reply:
@yusufsahinhamza thanks for clarifying this!

Reply:
Yes I confirm, we have to put '%1$s' (s = string).  The 's' should have disappeared during the translation.
I think there is a space missing after this code.

Reply:
@gilardh I didn't put a space there because there was no space before but maybe it's actually missed as you said.